src/Rule/ArgumentTypeMatchName.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of DOCtor-RST.
+ *
+ * (c) Oskar Stark <oskarstark@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Rule;
+
+use App\Value\Lines;
+use App\Value\RuleGroup;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ArgumentTypeMatchName extends AbstractRule implements LineContentRule, Configurable
```

Comment:
Have you a better name ?

`ArgumentNameMustMatchType` ?

Reply:
ArgumentVariableMustMatchType ?